File: sdk/response/response.go
package response

import (
	"net/http"

	"github.com/gin-gonic/gin"
)

func Success(c *gin.Context, data interface{}, msg string) {
	result(c, http.StatusOK, http.StatusOK, data, msg)
}
func Fail(c *gin.Context, data interface{}, msg string) {
	result(c, http.StatusOK, http.StatusBadRequest, data, msg)
}

func result(c *gin.Context, httpStatus int, code int, data interface{}, msg string) {
	c.JSON(httpStatus, gin.H{
		"code": code,
		"data": data,
		"msg":  msg})
}

File: llmops/utils/response.py
# * Copyright (c) KylinSoft  Co., Ltd. 2024.All rights reserved.
# * PilotGo-plugins licensed under the Mulan Permissive Software License, Version 2. 
# * See LICENSE file for more details.
# * Author: zhanghan2021 <zhanghan@kylinos.cn> 
# * Date: Thu Nov 7 15:53:47 2024 +0800 
from flask import jsonify


def success(data=None, msg=""):
    return result(200, 200, data, msg)


def fail(data=None, msg=""):
    return result(200, 400, data, msg)


def not_found(data=None, msg=""):
    return result(200, 404, data, msg)


def result(http_status, code, data=None, msg=""):
    response = {"code": code, "data": data, "msg": msg}
    return jsonify(response), http_status
